learn/using-the-openapi-tools.md

```diff
@@ -28,91 +28,103 @@ Code generation from OpenAPI to Ballerina can produce `ballerina mock services`
 
-## Client Stub from OpenAPI
+`(-o|--output)` is an optional parameter. You can give the output path for the generated files. 
+If not it will take the execution path as the output path.
 
+####Mock service from OpenAPI
 ```
-ballerina openapi gen-client [<moduleName>]:<clientName> 
-                   <openapi-contract> [-o <dir-path> | --output <dir-path>]
+ballerina openapi   -i <openapi-contract> --mode service
+                    [(-o|--output) outputFile]
 ```
-    
-Generates a Ballerina client stub for the service defined in a OpenAPI file.
-
-This client can be used in client applications to call the service defined in the OpenAPI file.
+Generates a Ballerina service for the OpenAPI file. This generated service is a mock version of the actual Ballerina service.
 
-## Service to OpenAPI Export
 
+####Client stub from OpenAPI
 ```
-ballerina openapi gen-contract [<moduleName>:]<serviceName> 
-                                [-i: <ballerinaFile> | --ballerina-file <ballerina-file>] 
-                                [-o: <openapi-contract> | --output <openapi-contract>] 
-                                [-s | --skip-bind]
+ballerina openapi   -i <openapi-contract> --mode client
+                    [(-o|--output) outputFile]
 ```
+Generates a Ballerina client stub for the service defined in a OpenAPI file. This client can be used in client 
+applications to call the service defined in the OpenAPI file.
 
-Export the Ballerina service to a definition of OpenApi Specification 3.0.
-For the export to work properly, the input Ballerina service should be defined using basic service and resource level HTTP annotations.
-
-## Client Stub for Service
-
-Generates a Ballerina client stub to communicate with a Ballerina service.
-
-All endpoint(s) that are used for client stub generation should be marked with the `@openapi:ClientEndpoint` annotation. If not, there might be errors during client stub generation. Endpoints that are not marked with this annotation are not picked for client stub generation.
-The `@openapi:ClientConfig { generate: true }` annotation is used to enable or disable client stub generation per service.
-
-## Samples
-
-### Mock Service from OpenAPI Sample
-
+###Ballerina to OpenAPI
+####Service to OpenAPI export
 ```
-ballerina openapi gen-service helloworld:helloService hello_service.yaml
+ballerina openapi   -i <ballerina file> 
+                    [(-o|--output) output openapi File]
 ```
+Export the Ballerina service to an  OpenApi Specification 3.0 definition. For the export to work properly, 
+the input Ballerina service should be defined using basic service and resource level HTTP annotations.
+If a user needs to document an OpenAPI contract for only one given service, then the user can use this command.
+``ballerina openapi -i <ballerina file> (-s | --service) <service name>``
```

Comment:
For Ballerina files, we need to use ```ballerina and for other commands, ```bash.

Please check all the other occurrences as well.